### PR TITLE
add tag & datasets

### PR DESCRIPTION
**Pull request by ruchim**
Adding 2 tags:
- bioinformatics
- research

### CODE REVIEW

**Comment by jflasher** [comment]
Hi @ruchim, thanks for the suggested tags. I know we don't provide a ton of guidance here, but `research` is probably a bit more vague than we'd like. `bioinformatics` seems fine, but are there datasets that you'd like to see tagged with that? If you want to add this tag to any appropriate datasets, we can look at pulling this in.

**Comment by ruchim** [comment]
Hey @jflasher 
I'm in the process of submitting applications for two datasets that contain references used in bioinformatics by human genome researchers.

**Comment by jflasher** [comment]
Great! Can you add the bioinformatics tag at the same time that you add those datasets using the tag?

**Comment by ruchim** [comment]
Adding yaml files for 2 datasets

File: datasets/gatk-test-data.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Name: GATK Test Data
+Description: GATK test data resource bundle is a collection of files for resequencing human genomic data with the GATK
+Contact: https://gatkforums.broadinstitute.org/gatk
+ManagedBy: Broad Institute
+UpdateFrequency: Every 3 months
+Tags:
+  - biology, bioinformatics, cancer, genetic, genomic
+License: N/A
+Resources:
```
[the remark this answers is not in the document]
Comment (wleepang) [reply datasets/gatk-test-data.yaml]: e.g.:
* arn
* region

File: datasets/broad-references.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Name: Broad Genome References
+Description: Broad maintained human genome reference builds hg19/hg38 and decoy references. 
+Contact: hensonc@broadinstitute.org
+ManagedBy: Broad Institute
+UpdateFrequency: Monthly
+Tags:
+  - biology, bioinformatics, cancer, genetic, genomic
```
Comment (wleepang) [inline datasets/broad-references.yaml]: break tags into a yaml list either:

```
tags: ["biology", "bioinformatics", ...]
```

or 

```
Tags:
- biology
- bioinformatics
- ...

**Comment by jedsundwall** [comment]
@ruchim for us to accept this PR, we need the following:

- Define a license for the data
- Add a URL to documentation for the data per the spec available at https://github.com/awslabs/open-data-registry/blob/master/README.md – this URL can simply be a link to an HTML file in the gatk-test-data bucket.
- Double-check the policy on the gatk-test-data bucket to make sure it's open. I cannot list the contents of the bucket as it exists now. It should have the following policy:

```json
{
  "Version": "2012-10-17",
  "Id": "gatk-test-data-pds-policy",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": "*",
      "Action": [
        "s3:List*",
        "s3:Get*"
      ],
      "Resource": [
        "arn:aws:s3:::gatk-test-data/*",
        "arn:aws:s3:::gatk-test-data"
      ]
    }
  ]
}
```

**Comment by ruchim** [comment]
- [x] Add license
- [x] Make both broad-references & gatk-test-data buckets public based on the policy listed above
- [x] HTTP url to docs

File: datasets/broad-references.yaml [hunk 3]
```diff
@@ -5,10 +5,14 @@ Contact: hensonc@broadinstitute.org
...
 Tags:
-  - biology, bioinformatics, cancer, genetic, genomics
+  - biology
+  - bioinformatics
+  - cancer
+  - genetic
+  - genomics
 License:  CC0 1.0 Universal (CC0 1.0) Public Domain Dedication
 Resources:
   - Description:
```
Comment (jflasher) [inline datasets/broad-references.yaml]: Just a note there needs to be a description filled in.

**Pull request merged**